src/Goteo/Library/Forms/Model/ProjectPostForm.php

```diff
@@ -37,7 +37,10 @@ public function createForm() {
                 'label' => 'regular-title',
                 'constraints' => array(
                     new Constraints\NotBlank(),
-                    new Constraints\Length(array('min' => 4)),
+                    new Constraints\Length([
+                        'min' => 4,
+                        'allowEmptyString' => false,
```

Comment:
When using `min` option in `Constraints\Length`, now the `allowEmptyString` option is required. Should it be true or false?